azure-iot-hub/tests/test_iothub_registry_module.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------
```

Comment:
>-------------------------------------------------------------------------- [](start = 2, length = 74)

This file contains tests that are already present in `test_iothub_resgitry_manager`. I first separated the device tests into `test_iothub_registry_device` and module tests into`test_iothub_registry_module` but later realized that I would not want to create another file for `get_service_stats` and `get_registry` type of tests. I am in support to delete this file unless anyone feels strongly.